packages/plugin-text/package.json

```diff
@@ -24,7 +24,7 @@
     "@types/ramda": "^0.26.1",
     "@types/react": "^16.8.0",
     "@types/react-dom": "^16.8.0",
-    "@types/slate": "0.47.4",
```

Comment:
was this by accident?

Reply:
`0.47.4` doesnt exist yet, so I changed it to the most recent `0.47.1`